techniques/system/inventory/1.0/fusionAgent.st

```diff
@@ -55,6 +55,14 @@ bundle agent computeInventoryTime
 
 bundle agent doInventory
 {
+  classes:
+    "disable_inventory" expression => fileexists("${g.rudder_disable_inventory_file}");
+  methods:
+    !disable_inventory::
+      "any" usebundle => doInventory_always;
+}
+bundle agent doInventory_always
```

Comment:
I don't think you should separate this check into a different bundle, because there is a purposely-designed class "force_inventory" that is checked below. Just like #993 allows to bypass the disable-agent file, it would be good if this force_inventory class also bypasses the disable-inventory file.

Therefore, it would make more sense to integrate this check into the logic below.